packages/dapp/src/app/profile/[slug]/Explorer.tsx

```diff
@@ -45,7 +51,13 @@ export const Explorer = ({
               key={`${chainId}-${endorsementId}`}
```

Comment:
We added new parameter `endorsementTab` so this needs to also change.

`${chainId}-${endorsementId}-${endorsementTab}`.